src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+| Cursor | Select a cursor design. |
+| Icons | Choose an icon set that displays custom icons for system tools and applications. |
+| Shell | Select or upload a shell theme. Using this option requires enabling `User Themes` in GNOME Extensions. |
+| Sound | Modify system feedback and notification sounds. |
+
+#### Background Options
+
+| Option | Function |
+|--------|----------|
+| Image | Select an image to set as the desktop background. |
+| Adjustment | Manipulate the appearance of the desktop background. Options include: <ul><li>Zoom</li><li>Centered</li><li>Scaled</li><li>Spanned</li><li>Stretched</li><li>Wallpaper |
+
+#### Lock Screen Options
+
+| Option | Function |
+|--------|----------|
+| Image | Select an image that will display at the lock screen. |
+| Adjustment | Manipulate the appearance of the lock screen background. Options include: <ul><li>Zoom</li><li>Centered</li><li>Scaled</li><li>Spanned</li><li>Stretched</li><li>Wallpaper |
+
+### Fonts
+
+`Fonts` settings allow you to specify the default fonts used throughout the operating system.
+
+![Fonts Tweaks](/images/gnome-tweaks-extensions/fonts-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Interface Text | Choose the font used for displayed text inside of a running application. |
+| Document Text | Choose the font used when creating a new text document. Applications like Libreoffice may override this selection. |
+| Monospace Text | Choose the font used in a Terminal session. This may also include coding applications and basic text editors. |
+| Legacy Window Titles | Choose the font used to display information in the top bar of running application windows. This is usually the name of the application. |
+| Hinting | Enable text hinting, which improves text readability for low-resolution displays. |
+| Antialiasing | Enable antialiasing, which improves text visibility when high-resolution text is displayed on a low-resolution screen. |
+| Scaling Factor | Enable text scaling, which increases font size for HiDPI displays while retaining text sharpness. |
+
+### Keyboard & Mouse
+
+`Keyboard & Mouse` settings extend customization of keyboard and mouse input behavior.
+
+![Keyboard Mouse Tweaks](/images/gnome-tweaks-extensions/keyboard-mouse-tweaks.png)
+
+#### Keyboard
+
+| Option | Function |
+|--------|----------|
+| Show Extended Input Sources | Display additional number of input sources found in the Settings application. |
+| Emacs Input | Apply Emacs keybindings throughout the operating system. |
+| Overview Shortcut | Choose whether left or right `SUPER` keys start the Launcher. |
+
+#### Mouse
+
+| Option | Function |
+|--------|----------|
+| Acceleration Profile | Choose how mouse movement speed adapts to user gestures. |
+| Pointer Location | Pressing the `Ctrl` key creates a visual notification at the mouse pointer location. |
+| Middle Click Paste | Clicking the middle mouse button pastes content from the clipboard. |
```

Comment:
It doesn't paste from the standard clipboard, it pastes whatever was last highlighted, known as the "primary selection," as opposed to the "clipboard selection" that's the normal clipboard.

(For future reference, this is an X11 feature, and I don't think anyone's implemented it in Wayland.)